src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/contactperson/ContactPersonService.kt

```diff
@@ -106,6 +106,16 @@ class ContactPersonService(private val personRepository: PersonRepository) {
           },
         )
       },
+      restrictions = it.restrictions.map { restriction ->
+        ContactRestriction(
+          id = restriction.id,
+          type = restriction.restrictionType.toCodeDescription(),
```

Comment:
Is description right here? Feels like a code...

Reply:
type? so type will be type.code = "BAN" type.description = "Banned"

so "type" refers to the "restriction" "type"